### PR TITLE
Add rabbitmq

### LINKED ISSUE #4: Support rabbitmq initialization

### PR DESCRIPTION
**Pull request by ilianakyl**
add rabbitmq to orka

resolves #4

### CODE REVIEW

File: examples/rabbitmq-example/config.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    appName: 'foo'
+  }, 
+  queue: {
+    prefetch: 100,
+    url: process.env.CLOUDAMQP_URL || 'amqp://localhost',
+    frameMax: 0x1000,
+    maxRetries: 0,
+    retryDelay: 1000,
+    connectDelay: 5000,
+    maxPriority: 10,
```
Comment (nikostoulas) [inline examples/rabbitmq-example/config.js]: rm

File: examples/rabbitmq-example/config.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    replyQueues: ['example']
```
Comment (nikostoulas) [inline examples/rabbitmq-example/config.js]: rm

File: package.json [hunk 3]
```diff
@@ -53,6 +54,8 @@
     "koa2-cors": "^2.0.6",
     "lodash": "^4.17.11",
     "log4js": "^4.2.0",
+    "newrelic": "^5.8.0",
```
Comment (nikostoulas) [inline package.json]: rm

File: examples/rabbitmq-example/routes.js [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+const ExampleHandler = require('./example-handler');
+
+const rabbit = getRabbit();
+
+new ExampleHandler('example_queue');
+
+module.exports = {
+  get: {
+    '/init': async (ctx, next) => {
+      // rabbit
```
Comment (alkiskal) [inline examples/rabbitmq-example/routes.js]: remove comments

File: src/initializers/rabbitmq/index.ts [hunk 5]
```diff
@@ -0,0 +1,40 @@
+import { Rabbit } from 'rabbit-queue';
+import * as Url from 'url';
+import { getLogger } from '../log4js';
+import { OrkaOptions } from '../../typings/orka';
+
+// const logger = getLogger('support.rabbit');
```
Comment (alkiskal) [inline src/initializers/rabbitmq/index.ts]: remove

File: src/initializers/rabbitmq/index.ts [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+const logger = getLogger('orka.errorHandler');
```
Comment (alkiskal) [inline src/initializers/rabbitmq/index.ts]: `orka.errorHandler` -> `orka.rabbit`

File: package.json (hunk 3 above)
Comment (alkiskal) [reply package.json]: No need, I'll take care of it when rebasing

Comment (nikostoulas) [reply package.json]: ❓

Comment (nikostoulas) [reply package.json]: What do you mean @alkiskal . It shouldn't be added again.

**Pull request merged**